client/controllers/locationSelect2.coffee

```diff
@@ -34,4 +35,5 @@ Template.locationSelect2.onRendered ->
               item: hit._source
             }
         }
-  @$(".select2-container").css("width", "100%")
+  #override the inline width of the select2 container and its text input field
+  $input.next(".select2-container").css("width", "100%").find("input").css("width", "100%")
```

Comment:
I did it this way because the widths are set directly on these elements as inline styles. I'm not sure if a css rule would override that.